metricbeat/module/apache/status/data.go

```diff
@@ -89,40 +125,7 @@ func eventMapping(body io.ReadCloser, hostname string) common.MapStr {
 	}
 
 	event := common.MapStr{
-		"hostname":          hostname,
-		"total_accesses":    h.ToInt("Total Accesses", fullEvent),
-		"total_kbytes":      h.ToInt("Total kBytes", fullEvent),
-		"requests_per_sec":  h.ToFloat("ReqPerSec", fullEvent),
-		"bytes_per_sec":     h.ToFloat("BytesPerSec", fullEvent),
-		"bytes_per_request": h.ToFloat("BytesPerReq", fullEvent),
-		"workers": common.MapStr{
-			"busy": h.ToInt("BusyWorkers", fullEvent),
-			"idle": h.ToInt("IdleWorkers", fullEvent),
-		},
-		"uptime": common.MapStr{
-			"server_uptime": h.ToInt("ServerUptimeSeconds", fullEvent),
-			"uptime":        h.ToInt("Uptime", fullEvent),
-		},
-		"cpu": common.MapStr{
-			"load":            h.ToFloat("CPULoad", fullEvent),
-			"user":            h.ToFloat("CPUUser", fullEvent),
-			"system":          h.ToFloat("CPUSystem", fullEvent),
-			"children_user":   h.ToFloat("CPUChildrenUser", fullEvent),
-			"children_system": h.ToFloat("CPUChildrenSystem", fullEvent),
-		},
-		"connections": common.MapStr{
-			"total": h.ToInt("ConnsTotal", fullEvent),
-			"async": common.MapStr{
-				"writing":    h.ToInt("ConnsAsyncWriting", fullEvent),
-				"keep_alive": h.ToInt("ConnsAsyncKeepAlive", fullEvent),
-				"closing":    h.ToInt("ConnsAsyncClosing", fullEvent),
-			},
-		},
-		"load": common.MapStr{
-			"1":  h.ToFloat("Load1", fullEvent),
-			"5":  h.ToFloat("Load5", fullEvent),
-			"15": h.ToFloat("Load15", fullEvent),
-		},
+		"hostname": hostname,
```

Comment:
it's not about it being a string, the hostname is different because it's not coming from the full event, that's why it's hard to put it in the schema. I considered defining the schema in the code rather then one-time, but creating all those objects to use only one time seemed wasteful. 

What we could do is put the hostname in the full event artificially and get it from there with the schema.